test/onnx/test_pytorch_onnx_onnxruntime.py

```diff
@@ -12587,6 +12580,30 @@ def forward(self, x):
         )
         self.run_test(model, input)
 
+    @skipIfUnsupportedMinOpsetVersion(10)
+    def test_qat_maxpool2d(self):
+        class M(torch.nn.Module):
```

Comment:
This could be made much less verbose with:
```python
model = torch.nn.Sequential(
    torch.quantization.QuantStub(),
    torch.nn.MaxPool2d(kernel_size=3, stride=2, padding=1),
    torch.quantization.DeQuantStub())
```